sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -728,6 +728,11 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
             {
               throw Azure::Core::RequestFailedException("error when reading body stream");
             }
+            if (chunk.Value.Details.ETag != etag)
+            {
+              throw Azure::Core::RequestFailedException(
+                  "file was modified in the middle of download");
```

Comment:
```suggestion
              throw Azure::Core::RequestFailedException(
                  "File was modified in the middle of download.");
```

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/2366